README.md

```diff
@@ -85,23 +85,23 @@ The following inputs can be used as `step.with` keys:
+| `driver`                   | String   | `docker-container` | Sets the [builder driver](https://docs.docker.com/engine/reference/commandline/buildx_create/#driver) to be used                                                             |
+| `driver-opts`              | List     |                    | List of additional [driver-specific options](https://docs.docker.com/engine/reference/commandline/buildx_create/#driver-opt) (eg. `image=moby/buildkit:master`)              |
+| `buildkitd-flags`          | String   |                    | [BuildKit daemon flags](https://docs.docker.com/engine/reference/commandline/buildx_create/#buildkitd-flags)                                                                 |
+| `buildkitd-config`¹        | String   |                    | [BuildKit daemon config file](https://docs.docker.com/engine/reference/commandline/buildx_create/#config)                                                                    |
+| `buildkitd-config-inline`¹ | String   |                    | Same as `buildkitd-config` but inline                                                                                                                                        |
+| `install`                  | Bool     | `false`            | Sets up `docker build` command as an alias to `docker buildx`                                                                                                                |
+| `use`                      | Bool     | `true`             | Switch to this builder instance                                                                                                                                              |
+| `endpoint`                 | String   |                    | [Optional address for docker socket](https://docs.docker.com/engine/reference/commandline/buildx_create/#description) or context from `docker context ls`                    |
+| `platforms`                | List/CSV |                    | Fixed [platforms](https://docs.docker.com/engine/reference/commandline/buildx_create/#platform) for current node. If not empty, values take priority over the detected ones. |
+| `append`                   | YAML     |                    | [Append additional nodes](docs/advanced/append-nodes.md) to the builder                                                                                                      |
+| `cache-binary`             | Bool     | `true`             | Cache buildx binary to GitHub Actions cache backend                                                                                                                          |
+| `cleanup`                  | Bool     | `true`             | Cleanup temp files and remove builder at the end of a job                                                                                                                    |
+
+> * ¹ `buildkitd-config` and `buildkitd-config-inline` are mutually exclusive
```

Comment:
GFM supports footnotes so we can use this syntax: 
https://gist.github.com/dvdksn/7ebcc3839ca66b25ee66c453c404cdb7

Reply:
Arg it puts the footnote after contributing section 😢 

https://github.com/crazy-max/docker-setup-buildx-action/tree/fix-inputs

Reply:
ah ok. Yeah I guess that makes sense in a way... "footnotes" 😆 If you don't want that you could just do something like

```
| foo  | bar |
|------|-----|
| a \* |     |

_\* lorem ipsum_
```

| foo  | bar |
|------|-----|
| a \* |     |

_\* lorem ipsum_